build.gradle

```diff
@@ -77,6 +77,11 @@ android {
         testInstrumentationRunnerArgument "TEST_USER", "\"$System.env.OCTEST_APP_USERNAME\""
         testInstrumentationRunnerArgument "TEST_PASSWORD", "\"$System.env.OCTEST_APP_PASSWORD\""
         testInstrumentationRunnerArgument "TEST_SERVER_URL", "\"$System.env.OCTEST_SERVER_BASE_URL\""
+
+        jackOptions {
+            enabled true
```

Comment:
Hi @davivel. This was added as the compiler was giving an error "Error:Jack is required to support java 8 language features. Either enable Jack or remove sourceCompatibility JavaVersion.VERSION_1_8".  More [here](http://stackoverflow.com/questions/37004069/errorjack-is-required-to-support-java-8-language-features).